### ISSUE TITLE
Windows 10 - All settings erased after update

### ISSUE THREAD

**Issue by KikoCosmetics** [getferdi/ferdi#2245]
**Bug description:**
Ferdi asked for a reboot to install an update. After reopening it was like I first installed. All empty.

**Steps to reproduce:**
Setup your services, then wait for an update and install it

**Expected behavior**
After an update I should see what I used to see before.

**Screenshots**
Useless I think

**Environment:**

- Operating System: From debug info i get win32, but it's Windows 10 Pro 64 bit
- Ferdi Version: 10.0.19042
- Server: Using without account

**Comment by vraravam** [comment]
Ferdi's in-app updater does not prompt for a reboot - this is some other app/malware on your system that can cause a *reboot*.

Also, as per #329 - the in-app updater is broken and awaiting a fix

**Comment by KikoCosmetics** [comment]
Test

**Comment by KikoCosmetics** [comment]
I had to open another issue, since it wouldn't let me post previously, if you just stopped closing this issue and listen what I have to say, maybe you could solve a bug.
Especially since I figured out what your bug is.

**Comment by vraravam** [comment]
> I had to open another issue, since it wouldn't let me post previously,

That might be a glitch in github. not something that we can control.

> Especially since I figured out what your bug is.

If you can provide the better description of the bug, I can reopen this ticket. Waiting.....

**Comment by vraravam** [comment]
(and btw, there's no Ferdi version with 10.x as the version number)

**Comment by KikoCosmetics** [comment]
Reboot/Restart in Italian is the same: "Riavvia". Anyway I meant reboot the application, not the entire system.

I was wrong about the version number, that one was a build number from the debug info.
Here's the real info:

![image](https://user-images.githubusercontent.com/93335982/142191819-4574ab22-17cd-46a8-96ba-8deab0978fb6.png)

Anyway the problem is that I'm using a portable version, but when I clicked the update button, the automatic setup didn't install a portable version, in fact when I uninstalled it, the portable installation (with all data) was left there. 

I really hope that you're not using the path to determine if the app is portable, because the issue may be that I put Ferdi under C:\Program Files\Ferdi although it's a portable installation.

**Comment by vraravam** [comment]
Thanks - that is better information. I will reopen this ticket and change the title to be more specific.

**Comment by vraravam** [comment]
This ticket needs to be fixed AFTER #329 is fixed unfortunately. Till then, there's no way to ensure that the in-app updater is not messing up some stuff during the update process.
cc @kytwb

**Comment by imtrobin** [comment]
I'm facing the same problem with portable version on win10. I replaced the old Ferdi-5.6.2.exe with the new Ferdi-5.6.3.exe with existing FerdiAppData folder. All the settings are lost. I was previously able to upgrade this way. When I start with the old Ferdi-5.6.2.exe, the settings are still intact.

**Comment by imtrobin** [comment]
So I tried 5.6.4, and it had the same issue. Upon investigating, the portable app now store the settings in another FerdiApp\FerdiAppData now instead of FerdiAppData

e.g

C:\Ferdi\Ferdi-5.6.4.exe
C:\Ferdi\FerdiAppData\FerdiAppData  (previously was C:\Ferdi\FerdiAppData)

Copying the existing FerdiData folder to the C:\Ferdi\FerdiAppData\FerdiAppData will restore previous setting.
